src/components/ArticlePage.jsx

```diff
@@ -43,7 +59,37 @@ const ArticlePage = () => {
     }
   };
 
-  console.log(comments);
+  const handleCommentSubmit = (e) => {
+    e.preventDefault();
+    setIsLoadingComment(true);
+    if (commentInput === "") {
+      setNoCommentInput(true);
+      setIsLoadingComment(false);
```

Comment:
If the above is so you don't try and post an empty comment, another way of dealing with this would be to just add required to your input. That would stop you from trying to submit